docs/hub/models-main.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Models
+---
+
+<h1>Models</h1>
+
+The Hugging Face Hub hosts many models for a [variety of machine learning tasks](https://huggingface.co/tasks). Models are stored in repositories, so they benefit from [all the features](./repositories-main) possessed by every repo on the Hugging Face Hub. Additionally, model repos have attributes that make exploring and using models as easy as possible. These docs will take you through everything you'll need to know to find models on the Hub, upload your models, and make the most of everything the Model Hub offers!
+
+## Contents
+
+- [The Model Hub](./models-the-hub)
+- [Model Cards](./models-cards)
+  - [CO<sub>2</sub> emissions](./models-cards-co2)
+- [Tasks](./models-tasks)
+- [Interacting with models on the Hub](./models-interacting)
```

Comment:
`Interacting with models` sounds vague to me, WDYT of changing to something like `downloading and uploading models to/from the hub`?

or even split into `download` and `upload`

Reply:
Sounds good to me! I put that in my to-do list for the final polishing pass :)